### PR TITLE
Xkwoad

### PR DESCRIPTION
**Pull request by nichlas0010**

### CODE REVIEW

File: models/Byond/Endpoints/EndpointServerStatus.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ class EndpointASayMessage extends APIEndpoint {
       var embedmessage = "<@&" + config.discord_subscriber_role + ">";
 
       for (var channel of discord.getPrimaryGuild().channels.array()) {
-        if (channel.id == config.discord_public_channel) {
```
Comment (AsV9) [inline models/Byond/Endpoints/EndpointServerStatus.js]: 👎

File: models/Github/GithubManager.js [hunk 2]
```diff
@@ -140,7 +140,7 @@ class GithubManager {
     }
 
     for (var channel of discordSubsystem.getPrimaryGuild().channels.array()) {
-      if (channel.id == config.discord_coder_channel || channel.id == config.discord_coder_public_channel) {
+      if (channel.id == config.discord_botspam_channel) {
```
Comment (AsV9) [inline models/Github/GithubManager.js]: 👍

File: models/Byond/Endpoints/EndpointServerStatus.js [hunk 3]
```diff
@@ -40,6 +40,9 @@ class EndpointASayMessage extends APIEndpoint {
         if (channel.id == config.discord_public_channel) {
           channel.sendEmbed(embed, embedmessage);
         }
+        if(channel.id == config.discord_botspam_channel) {
```
Comment (AshCorr) [inline models/Byond/Endpoints/EndpointServerStatus.js]: Don't need to post it in both public and botspam, that just makes it annoying for subscribers.

Comment (nichlas0010) [reply models/Byond/Endpoints/EndpointServerStatus.js]: you do see how embedmessage isn't included in the one in botspam, right?

Comment (nichlas0010) [reply models/Byond/Endpoints/EndpointServerStatus.js]: like, that's the reason I added another if instead of just doing ||

Comment (AshCorr) [reply models/Byond/Endpoints/EndpointServerStatus.js]: ah, my bad

**Pull request merged**